gpytorch/mlls/leave_one_out_likelihood.py

```diff
@@ -0,0 +1,66 @@
+import math
+
+import torch
+from torch import Tensor
+
+from ..distributions import MultivariateNormal
+from .exact_marginal_log_likelihood import ExactMarginalLogLikelihood
+
+
+class LeaveOneOutLikelihood(ExactMarginalLogLikelihood):
```

Comment:
could call this `LeaveOneOuPseudoLikelihood` if we wanted to be precise

Reply:
Good suggestion, will do!